api/agentpb/agent.proto

```diff
@@ -120,6 +120,9 @@ message QueryActionResult {
 
 // StartActionRequest is a ServerMessage asking pmm-agent to start action.
 message StartActionRequest {
+  // PTSummaryParams describes parameters for PT summary.
+  message PTSummaryParams {}
```

Comment:
Please move it after MongoDBExplainParams so the order is the same

Reply:
Order fixed.